loader-tests/check-opml.ts

```diff
@@ -10,7 +10,8 @@ import {
   finish,
```

Comment:
We need progress bar also in `test-loaders`

Reply:
done